src/index.ts

```diff
@@ -59,54 +69,47 @@ app.use(morgan('tiny'))
 app.use('/api/v1/', router)
 const web3 = new Web3(argv['ethereum-node-url'] as string)
 
+const poolOptions = {
+  minWorkers: argv['num-threads'],
+  maxWorkers: argv['num-threads'],
+  maxQueueSize: argv['max-queue-size'],
+}
+export const pool = workerpool.pool(path.join(__dirname, '../build/worker.js'), poolOptions)
```

Comment:
I don't know and I don't know what the typescript way is. This line originally comes from Felix. Since it works I would like to leave it like this unless you know the correct way to do it.